doc/en/user/source/community/ogc-api/features/index.rst

```diff
@@ -0,0 +1,202 @@
+OGC API Features
+================
+
+An `OGC Features API <https://github.com/opengeospatial/ogcapi-features>`_ up to date with the  released version of the Features API, along with implementation of the `draft CQL filtering extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/cql>`_ and the `draft multi-crs extension <https://github.com/opengeospatial/ogcapi-features/tree/master/extensions/crs>`__.
+
+Service
+-------
+
+The OGC API Features Service is accessed via the :guilabel:`FEATURES` version :guilabel:`1.0` link on the home page.
+
+Capabilities
+''''''''''''
+
+The service is self described using:
+
+* ``html``: A collection of web pages, with links for navigation between content (and that can be indexed by search engines for discoverability).
+
+  .. figure:: img/features.png
+ 
+     OGC API Features service
+
+* `application/json`: A collection :file:`json` documents, with reference between each document for programatic access by web developers.
+
+  .. code-block:: json
+  
+     {
+       "title": "GeoServer Web Feature Service",
+       "description": "This is the reference implementation of WFS 1.0.0 and WFS 1.1.0, supports all WFS operations including Transaction.",
+       "links": [
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=application%2Fx-yaml",
+           "rel": "alternate",
+           "type": "application/x-yaml",
+           "title": "This document as application/x-yaml"
+         },
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=application%2Fjson",
+           "rel": "self",
+           "type": "application/json",
+           "title": "This document"
+         },
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=text%2Fhtml",
+           "rel": "alternate",
+           "type": "text/html",
+           "title": "This document as text/html"
+         },
+         ...
+
+* ``application/x-yaml``: A collection of :file:`yaml` documents, with references between each document for programatic access.
+ 
+  .. code-block:: yaml
+  
+     title: GeoServer Web Feature Service
+     description: This is the reference implementation of WFS 1.0.0 and WFS 1.1.0, supports
+       all WFS operations including Transaction.
+     links:
+     - href: http://localhost:8080/geoserver/ogc/features/?f=application%2Fx-yaml
+       rel: self
+       type: application/x-yaml
+       title: This document
+     - href: http://localhost:8080/geoserver/ogc/features/?f=application%2Fjson
+       rel: alternate
+       type: application/json
+       title: This document as application/json
+     - href: http://localhost:8080/geoserver/ogc/features/?f=text%2Fhtml
+       rel: alternate
+       type: text/html
+       title: This document as text/html
+
+The service title and description are provided by the existing :ref:`wfs` settings.
```

Comment:
Perhaps orthogonal to this PR, but maybe we can update the generic WFS words too if we're going to keep using them for other things.

Reply:
Agreed, I think we can do so when ogcapi graduates, unless we should  do so as part of this sprint.